backend/config.py

```diff
@@ -429,6 +429,9 @@ def statuses(self) -> List:
 slack_bot_token = os.getenv("SLACK_BOT_TOKEN")
 slack_user_token = os.getenv("SLACK_USER_TOKEN")
 
+# Enable/Disable chatter message in incident channel
+slack_chatter_message_enabled = os.getenv("SLACK_CHATTER_MESSAGE_ENABLED", default="true")
```

Comment:
Would you mind adding this to the `config.yaml` section instead? It could go under `.options`. Here's an example of another contribution that made changes to this file. https://github.com/echoboomer/incident-bot/pull/397/files#diff-651976572b51d1a16ede6a1c4c1be9a1482605ce57ca58fbfca25e4a55c43210